pyproject.toml

```diff
@@ -34,6 +34,7 @@ dev = [
     "kolibri2zim[scripts]",
     "kolibri2zim[lint]",
     "kolibri2zim[check]",
+    "hatchling",
```

Comment:
This should not be necessary. hatchling is a build dependency (already declared as such)

Reply:
This is necessary for [this line](https://github.com/openzim/kolibri/blob/2c16e5f261cebfcf34a4ddf78180c8102210f9a4/hatch_build.py#L5) to work in my dev environment (otherwise it is not recognized). 
I don't know exactly why, but without this line `hatch` does not add `hatchling` as a dependency of the `dev` environment, and it is hence not available (and my code editor complains, I can't benefit from code suggestions, ...)

Reply:
Well this hook is a building hook so it is to be imported only at build time where hatchling is installed because it's the build backend and referenced properly.

- If you want to run it as a standalone script (not sure how), then you may need it.
- Your text editor surely can't recognize it if it's not installed.

It's not a dev dependency but we surely don't need a `build` feature so let's keep it if it eases devel but add a comment above so we know it's only for hook development.